<launch>
    <node name='speechResponse' pkg='bdbd' type='speechResponse' ns='bdbd' output='screen'/>
</launch>
